variablegroups/AAT-AKS-Common.json

```diff
@@ -46,7 +46,7 @@
       "value": "[\"10.96.139.254\"]"
     },
     "Subscription.keyvaultName": {
-      "value": "cftapps-stg"
+      "value": "cftapps-$(Env.Override.Name)"
```

Comment:
keyvault name? is this correct. I thought we have a different keyvault name in prod.